lib/orogen/gen/project.rb

```diff
@@ -48,11 +48,16 @@ class << self
             attr_accessor :relative_generation_directory
         end
         @relative_generation_directory = nil
-
+        
         def self.orocos_target=(target)
             @orocos_target = target.to_s
         end
-
+           
+        def self.enable_namespace=(value); @disabled_namespaces.delete(value) end
```

Comment:
This is why some people hate operator overloading. You make assignation '=' behave resp. like << and delete. Yuk.